### DISCUSSION TITLE
Difficulty reopening closed chat

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#1893] [General]
Hi. It is complicated for users to reopen a closed chat:
- On a closed chat the X Close chat still appears on the right panel and there is no Reopen chat option.
- When changing the status using the i icon to Active , the message textarea no longer shows the "chat is closed" message but still one cannot send messages
- It is necessary to reload the page to be able to chat normally.

**Comment by remdex** [comment]
Why user should be reopening closed chat at first place?

**Reply by alonsoac** [reply]
There are many reasons for this depending on the company that uses the software. For example new information may be available after a chat was closed and customer needs to be contacted again as a continuation of the same chat over Whatsapp or Facebook.

**Reply by remdex** [reply]
Can you write down your workflow in the following order

1. Customer comes to website
2. Starts a chat
....

**Reply by alonsoac** [reply]
1. Customer writes via Whatsapp, new chat is created
2. Agent services customer, thinks case is done and closes chat
3. New information is available and customer needs to be contacted again
4. Agent reopens chat by changing status and reloading page
5. Agent now can continue chat with customer

**Reply by remdex** [reply]
This above workflow scenario should not be applied to regular chats. If you write a message to already closed chat, it's for sure won't popup for visitors. You can enable that if a customer starts show previous conversation message
https://doc.livehelperchat.com/docs/theme/#show-start-chat-form-instantly-after-operator-closes-a-chat

Or operator can click invite to chat just
![image](https://user-images.githubusercontent.com/1146085/214523962-4e52fa50-2df6-4906-87bf-44ff89866bdb.png)

**Reply by alonsoac** [reply]
Sure this is not for regular chats. The user would know that is it is a Whatsapp or FB or Instagram chat and that this chat can be continued transparently for the customer, so reopening the chat is the correct action.  I think the future of LHC is in these newer chat mediums.

**Comment by alternativas10** [comment]
Hello. I have the same problem. When I try to open a changing the status, conversation generated by WhatsApp and there is a cron active, it immediately closes the conversation.


How can I reopen a conversation. I tried changing the status, changing to pending... but it's not possible, it closes immediately.

<img width="467" alt="Captura de pantalla 2023-03-08 a la(s) 3 42 03 p m" src="https://user-images.githubusercontent.com/37350883/223845085-6278746d-dc5e-477c-8a81-089e8f84605d.png">

**Comment by remdex** [comment]
Can you share your close chat configuration section? I would try to see that scenario with third party chats FB it whatsapp

**Comment by remdex** [comment]
Please try this change - https://github.com/LiveHelperChat/livehelperchat/commit/75da20ae4b789ad387d326147586ee7c12f2143e
Chats should be reopened in active state. 

If chats still will be closed, try to experiment what closing cronjobs you have :)
